src/pages/index.js

```diff
@@ -1,7 +1,5 @@
 import React from "react";
-import Link from "gatsby-link";
-import styled, { keyframes } from "styled-components";
-import { injectGlobal } from 'styled-components';
```

Comment:
Why was even done this way?

Reply:
Earlier Link was available in gatsby-link but now it can be imported from gatsby directly. Plus we are not using Link anymore anyway I guess it crawled into the project because base of this project was copied from last project which had separate leaderboard page. As for separate import of components from styled-components it is simply the result of addition of injectGlobal later in the project in my guess.